website/docs/r/one_dashboard.html.markdown

```diff
@@ -169,6 +241,12 @@ All nested `widget` blocks support the following common arguments:
   * `width` - (Optional) Width of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `4`.
   * `height` - (Optional) Height of the widget.  Valid values are `1` to `12` inclusive.  Defaults to `3`.
   * `ignore_time_range` - (Optional) With this turned on, the time range in this query will override the time picker on dashboards and other pages. Defaults to `false`.
+  * `facet_show_other_series` - (Optional) With this turned on, this customization allows you to have the Other groups option visible or not if a facets ona query returns are more than 2000 for bar charts, pie charts, tables, the Other groups facet aggregates the rest of the facets. Defaults to `false`
```

Comment:
(Optional) Enable or disable the `Other` group in visualisations. The other group is used if a facet on a query returns more than 2000 items for bar charts, pie charts, and tables. The other group aggregates the rest of the facets. Defaults to `false`

Reply:
updated